examples/fine-tuning.ts

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env -S npm run tsn -T
+
+/**
+ * Fine-tuning allows you to train models on your own data.
+ *
+ * See these guides for more information:
+ * - https://help.openai.com/en/articles/5528730-fine-tuning-a-classifier-to-improve-truthfulness
```

Comment:
I don't think this is up to date @simonpfish we may want to remove it

Reply:
Oh you're right, good catch, I'll link to the new one.

Reply:
Oh I see that you already updated it, nice :)